spec.md

```diff
@@ -311,7 +315,7 @@ message GetSupportedVersionsResponse {
   }
 }
 
-// Specifies the version in Semantic Version 2.0 format.
+// Specifies a version in Semantic Version 2.0 format.
```

Comment:
Hi @jdef,

Perhaps link to the SemVer site?

`// Specifies a version in Semantic Version 2.0 (http://semver.org) format.`